fs_test.go

```diff
@@ -33,7 +33,7 @@ func TestNewFS(t *testing.T) {
 func getProcFixtures(t *testing.T) FS {
 	fs, err := NewFS(procTestFixtures)
 	if err != nil {
-		t.Fatal(err)
+		t.Errorf("Creating pseudo fs from getProcFixtures failed at fixtures/proc")
```

Comment:
Should probably include the original error in the message string to make it easier to tell what is the root cause.

Reply:
Got it.